apps/src/templates/teacherDashboard/ParentLetter.jsx

```diff
@@ -97,34 +100,26 @@ class ParentLetter extends React.Component {
               studentName: studentName,
             })}
           />
-          <h1>{i18n.parentLetterStep1()}</h1>
-          <SafeMarkdown
-            markdown={i18n.parentLetterStep1Details({
-              engagementLink: ENGAGEMENT_URL,
-              videosLink: pegasus(`/educate/resources/videos`),
-            })}
-          />
-          <h1>{i18n.parentLetterStep2()}</h1>
-          <SignInInstructions
-            loginType={loginType}
-            secretPicturePath={secretPicturePath}
-            secretWords={secretWords}
-            sectionCode={sectionCode}
-            studentName={studentName}
-          />
-          <SafeMarkdown
-            markdown={i18n.parentLetterStep2Details({
-              studentName: studentName,
-              projectsLink: studio('/projects/public'),
-              atHomeLink: pegasus('/athome'),
-            })}
-          />
-          <h1>{i18n.parentLetterStep3()}</h1>
-          <SafeMarkdown
-            markdown={i18n.parentLetterStep3Details({
-              accountEditLink: studio('/users/edit'),
-            })}
-          />
+          {loginType !== SectionLoginType.lti_v1 ? (
+            <RegularParentLetterSteps
+              loginType={loginType}
+              loginTypeName={loginTypeName}
+              secretPicturePath={secretPicturePath}
+              secretWords={secretWords}
+              sectionCode={sectionCode}
+              studentName={studentName}
+            />
+          ) : (
+            <LMSParentLetterSteps
+              loginType={loginType}
+              loginTypeName={loginTypeName}
+              secretPicturePath={secretPicturePath}
+              secretWords={secretWords}
+              sectionCode={sectionCode}
+              studentName={studentName}
+            />
+          )}
```

Comment:
Just a minor nit (no need to accept). You can create a function `getParentLetterSteps` and instead of the ternary operator perhaps a switch statement may be slightly cleaner:

```
switch(loginType) {
  case SectionLoginType.lti_v1: return <LMSParentLetterSteps loginType{loginType} ... />
  default: <RegularParentLetterSteps loginType={loginType} .../>
}
```